packages/browser-destinations/src/destinations/braze/logCustomEvent/index.ts

```diff
@@ -14,18 +15,24 @@ const action: BrowserActionDefinition<Settings, typeof appboy, Payload> = {
       required: true,
       description: 'The identifier for the event to track.',
       label: 'eventName',
-      default: 'event'
+      default: {
+        '@path': '$.event'
+      }
     },
     eventProperties: {
       type: 'object',
       required: false,
       description: 'Hash of properties for this event.',
       label: 'eventProperties',
-      default: {}
+      default: {
+        '@path': '$.properties'
+      }
     }
   },
   perform: (client, event) => {
-    client.logCustomEvent(event.payload.eventName, event.payload.eventProperties)
+    const reservedKeys = Object.keys(action.fields.eventProperties.properties ?? {})
```

Comment:
This doesn't make a ton of sense to me. Should this be: 
```suggestion
    const reservedKeys = Object.keys(action.fields ?? {})
```

Reply:
Oh I guess it honestly doesnt matter since there are only two keys here. I would remove the `omit` stuff since there's really no point for this action. I thought there were more top-level fields originally, sorry!